RiotSwiftUI/Modules/LocationSharing/MapView/View/LocationSharingMapView.swift

```diff
@@ -75,14 +81,18 @@ struct LocationSharingMapView: UIViewRepresentable {
         mapView.vc_removeAllAnnotations()
         mapView.addAnnotations(annotations)
         
-        if let highlightedAnnotation = highlightedAnnotation, !showsUserLocation {
-            mapView.setCenter(highlightedAnnotation.coordinate, zoomLevel: Constants.mapZoomLevel, animated: false)
+        if let highlightedAnnotation = highlightedAnnotation, showsUserLocationMode != .follow {
```

Comment:
Is it correct with enter the if also with `.hide`?

Reply:
yes it is, it's the current user own location that is .hide, not the the others sharing locations